### ISSUE TITLE
Error `Unexpected X11 event: 65` when triggering keynav

### ISSUE THREAD

**Issue by sim590** [jordansissel/keynav#51]
At some point, which I can't describe with more details, `keynav` will cause the keyboard to freeze with the error `Unexpected X11 event: 65`. I have to kill `keynav` (by jumping in an old-school TTY terminal) in order to get the control of the keyboard back. Once it happens a first time, there's nothing to do. Every time keynav will be restarted and triggered by the appropriate keystroke, the same thing will happen. There nothing to do other than restarting X in order to "fix it" for a while. I don't know how to reproduce for now.

**Comment by sitepodmatt** [comment]
Seeing this in i3wm too

**Comment by eyalev** [comment]
Happens to me when the keyboard is on a Hebrew layout instead of English

**Comment by sitepodmatt** [comment]
Im on en_US keyboard and it happens to me at launch Unexpected X11 event: 65` after the targetter is overlaid, hjlk are useless at point of launch.  It may have something to do with switching to regolith linux in last few days so my compositor has changed picom.

**Comment by exAClior** [comment]
Using manjaro Linux with xmonad + picom, seeing this too

**Comment by sitepodmatt** [comment]
Killing picom it seems to work. So I think it is a compositor problem.

**Comment by PaddiM8** [comment]
Yeah picom seems to be the problem for me as well. Does anyone know how to get around this?

**Comment by sawyer07** [comment]
Is this the same issue as https://github.com/jordansissel/keynav/issues/42 ? Commenting out `XUnmapWindow(dpy, zone);` in keynav.c resolved this for me. I never had to kill keynav from TTY though. I'd lose control of keyboard but in reality the escape key sufficed to close the program; the grid was simply not displaying. Commenting out the line in question yielded the same result for me as killing picom did. Thanks!

**Comment by aflyingpumpkin** [comment]
If you kill picom, start keynav, then re-open picom it works as intended.

**Comment by aflyingpumpkin** [comment]
Keynav is great!
